locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_("")
```

Comment:
This is probably caused by https://github.com/theforeman/foreman_remote_execution/blob/master/app/lib/actions/remote_execution/run_host_job.rb#L63, I had to comment it out to get the rake task to run, otherwise I get this error:

```
NoMethodError: undefined method `[]' for nil:NilClass
/home/vagrant/foreman_remote_execution/app/lib/actions/remote_execution/run_host_job.rb:63:in `humanized_name'
```